src/overlays/actors/ovl_En_Maruta/z_en_maruta.c

```diff
@@ -58,7 +58,40 @@ u8 D_80B386C0[] = {
 };
 
 s32 D_80B386CC[] = {
-    5, 5, 3, 3, 7, 7, 7, 7, 3, 3, 3, 3, 7, 7, 3, 3, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 7, 7, 7,
+    /* PLAYER_MWA_FORWARD_SLASH_1H */ 5,
+    /* PLAYER_MWA_FORWARD_SLASH_2H */ 5,
+    /* PLAYER_MWA_FORWARD_COMBO_1H */ 3,
+    /* PLAYER_MWA_FORWARD_COMBO_2H */ 3,
+    /* PLAYER_MWA_RIGHT_SLASH_1H */ 7,
```

Comment:
```suggestion
    5, // PLAYER_MWA_FORWARD_SLASH_1H
    5, // PLAYER_MWA_FORWARD_SLASH_2H
    3, // PLAYER_MWA_FORWARD_COMBO_1H 
    3, // PLAYER_MWA_FORWARD_COMBO_2H
    7, // PLAYER_MWA_RIGHT_SLASH_1H
```
These should be stylized like this: